generator/otel-config.yaml

```
@@ -0,0 +1,26 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: localhost:4317
+      http:
+        endpoint: localhost:4318
+
+processors:
+  batch:
+    timeout: 1s
+    send_batch_size: 1024
+
+exporters:
+  otlphttp/logs:
+    # change this depending on where you're running loki (can be made configurable)
+    endpoint: http://localhost:3100/otlp
```

`yarn server` will not produce any otel logs as it is using `localhost` by default while Loki does not run in the same container.

Should we default to 
```suggestion
    endpoint: http://host.docker.internal:3100/otlp
```
or make it configurable?

---

sure that makes sense. thanks!

---

i think better to make it configurable somehow, i need to fix the dockerfile after this commit to use `localhost`.. will push the commit in sometime